src/transforms/geoip.rs

```diff
@@ -287,11 +347,16 @@ mod tests {
         let mut exp_geoip_attr = HashMap::new();
         exp_geoip_attr.insert("city_name", "");
         exp_geoip_attr.insert("country_code", "");
+        exp_geoip_attr.insert("country_name", "");
+        exp_geoip_attr.insert("continent_code", "");
+        exp_geoip_attr.insert("region_code", "");
+        exp_geoip_attr.insert("region_name", "");
         exp_geoip_attr.insert("continent_code", "");
         exp_geoip_attr.insert("timezone", "");
         exp_geoip_attr.insert("latitude", "");
         exp_geoip_attr.insert("longitude", "");
         exp_geoip_attr.insert("postal_code", "");
+        exp_geoip_attr.insert("metro_code", "<null>");
```

Comment:
It seems odd that this one is inserted as `<null>` while others are empty strings. I know you call this out in the PR description as probably the better solution, but should we be consistent here?